### DISCUSSION TITLE
Overseerr install error

### DISCUSSION THREAD

**Discussion by slashdelta** [tteck/Proxmox#2024] [Help Needed] [answered]
Trying to install the Overseerr container, but I'm getting an error

> Using Default Settings
> Using Distribution: debian
> Using debian Version: 12
> Using Container Type: 1
> Using Root Password: Automatic Login
> Using Container ID: 102
> Using Hostname: overseerr
> Using Disk Size: 8GB
> Allocated Cores 2
> Allocated Ram 2048
> Using Bridge: vmbr0
> Using Static IP Address: dhcp
> Using Gateway Address: Default
> Disable IPv6: No
> Using Interface MTU Size: Default
> Using DNS Search Domain: Host
> Using DNS Server Address: Host
> Using MAC Address: Default
> Using VLAN Tag: Default
> Enable Root SSH Access: No
> Enable Verbose Mode: No
> Creating a Overseerr LXC using the above default settings
>  ✓ Using local for Template Storage.
>  ✓ Using local-zfs for Container Storage.
>  ✓ Updated LXC Template List
>  ✓ LXC Container 102 was successfully created.
>  ✓ Started LXC Container
>  ✓ Set up Container OS
>  ✓ Network Connected: 192.168.32.38 dd80:53c3:1058:0:d49f:e9ff:feba:e8aa 
>  ✓ Internet Connected
>  ✓ DNS Resolved github.com to 140.82.113.3
>  ✓ Updated Container OS
>  ✓ Installed Dependencies
>  ✓ Set up Node.js Repository
>  ✓ Installed Node.js
>  - Installing Yarn...
> [ERROR] in line 24: exit code 1: while executing command "$@" > /dev/null 2>&1
> The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.

Happens regardless of default settings or Advanced with different options

Any ideas on the error and how to fix it?

**Comment by tteck** [comment]
- When encountering the error message [ERROR] in line 24: exit code *: while executing command "$@" > /dev/null 2>&1, make sure to run the script in verbose mode to accurately determine the underlying issue.

**Comment by tteck** [comment]
The script runs without any problems from this end. 

```
   ____
  / __ \_   _____  _____________  ___  __________
 / / / / | / / _ \/ ___/ ___/ _ \/ _ \/ ___/ ___/
/ /_/ /| |/ /  __/ /  (__  )  __/  __/ /  / /
\____/ |___/\___/_/  /____/\___/\___/_/  /_/

Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 112
Using Hostname: overseerr
Using Disk Size: 8
Allocated Cores: 2
Allocated RAM: 2048
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Overseerr LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.253 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.114.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up Node.js Repository
 ✓ Installed Node.js
 ✓ Installed Yarn
 ✓ Installed Overseerr
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Overseerr should be reachable by going to the following URL.
         http://192.168.86.253:5055 
```

**Comment by slashdelta** [comment]
Whoops, i rip through the install prompts, forgot about verbose

> >  ✓ Installed Node.js
> + msg_info 'Installing Yarn'
> + local 'msg=Installing Yarn'
> + echo -ne ' - \033[33mInstalling Yarn...'
>  - Installing Yarn...+ npm install -g yarn
> npm ERR! code ENETUNREACH
> npm ERR! syscall connect
> npm ERR! errno ENETUNREACH
> npm ERR! request to https://registry.npmjs.org/yarn failed, reason: connect ENETUNREACH 2606:4700::6810:1e22:443
> 
> npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2023-11-10T23_56_17_809Z-debug-0.log
> ++ error_handler 37 '$STD npm install -g yarn'
> ++ local exit_code=1
> ++ local line_number=37
> ++ local 'command=$STD npm install -g yarn'
> ++ local 'error_message=\033[01;31m[ERROR]\033[m in line \033[01;31m37\033[m: exit code \033[01;31m1\033[m: while executing command \033[33m$STD npm install -g yarn\033[m'
> ++ echo -e '\n\033[01;31m[ERROR]\033[m in line \033[01;31m37\033[m: exit code \033[01;31m1\033[m: while executing command \033[33m$STD npm install -g yarn\033[m'
> 
> [ERROR] in line 37: exit code 1: while executing command $STD npm install -g yarn
> ++ [[ 37 -eq 24 ]]
> + exit

**Reply by tteck** [reply]
The error you're encountering (`ENETUNREACH`) indicates that your machine is unable to reach the npm registry, possibly due to network issues.

**Reply by tteck** [reply]
Ensure that your firewall or antivirus/ad-blocker software is not blocking npm or the connection to the npm registry.

**Comment by slashdelta** [comment, marked as answer]
Ha, thanks to you pointing out networking, it looked like IPv6 so...
I disabled IPv6 option on the install script, worked like a charm

Thanks for the assist!
